src/main/scala/chisel3/stage/package.scala

```diff
@@ -12,6 +12,8 @@ import scala.annotation.nowarn
 
 package object stage {
 
+  final val pleaseSwitchToCIRCT = deprecatedMFCMessage + "Please switch to circt.stage.ChiselStage."
```

Comment:
```suggestion
  final val pleaseSwitchToCIRCT = deprecatedMFCMessage + " Please switch to circt.stage.ChiselStage."
```
Warning currently runs together:
```
object ChiselStage in package stage is deprecated (since Chisel 3.6): this feature will not be supported as part of the migration to the MLIR-based FIRRTL Compiler (MFC). For more information about this migration, please see the Chisel ROADMAP.md.Please switch to circt.stage.ChiselStage.
```

Reply:
Thanks! Good catch. Corrected in the last `!fixup` commit.